ejabberd_bridge/management/commands/ejabberd_auth.py

```diff
@@ -47,33 +49,37 @@ def to_ejabberd(self, answer=False):
         sys.stdout.flush()
 
     def auth(self, username=None, server="localhost", password=None):
```

Comment:
this make me confuse sometime because the username here is actually a user_id. Probably it's better to call id user_id instead?

Reply:
yes true @orhan89 , will fix it